packages/connect-agreement/jest.config.js

```diff
@@ -0,0 +1,15 @@
+const baseConfig = require('../../jest.config.base')
+
+const packageName = require('./package.json').name.split('@aragon/').pop()
+
+module.exports = {
+  ...baseConfig,
+  roots: [`<rootDir>/packages/${packageName}`],
+  collectCoverageFrom: ['src/**/*.{ts,tsx}'],
+  testRegex: `(packages/${packageName}/.*/__tests__/.*|\\.(test|spec))\\.tsx?$`,
+  modulePaths: [`<rootDir>/packages/${packageName}/src/`],
+  name: packageName,
+  displayName: 'AGREEMENT',
+  testTimeout: 30000,
```

Comment:
I’m wondering if this is still needed, now that we have a disconnection mechanism?